nubilanding/src/components/nosotros/styles.module.scss

```diff
@@ -0,0 +1,58 @@
+.contenedor {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 50px;
+}
+
+.contenedorTitulo {
+  background-image: url(../../assets/bg_main_about_us.png);
+  background-size: 117% 90%;
+  background-repeat: no-repeat;
+  text-align: center;
+  width: 100%;
+  height: 435px;
+
+  h1 {
+    padding: 130px;
+    font-family: "Rubik";
+    font-style: italic;
+    font-size: 48px;
+    font-weight: 600;
+    color: #214960;
```

Comment:
Mover colores a un archivo `_colors.scss` es una buena practica. Los nombres de los colores podrian salir de acá: http://chir.ag/projects/name-that-color.

Reply:
y esta sugerencia la tenemos pendiente de realizarla pero estamos en proceso!!